index.bs

```diff
@@ -5822,6 +5832,11 @@ Records must not be used as the type of an [=attribute=] or
 The [=type name=] of a record type is the concatenation of the type
 name for |K|, the type name for |V| and the string “Record”.
 
+Any [=ordered map=] can be implicitly treated as a record, as long as it contains only
```

Comment:
Similar to sequence, `as a `record<K, V>` might be clearer, and might let you omit the "Additionally..." sentence since the restriction on *K* was already stated above.